documentation/book/cluster-operator.adoc

```diff
@@ -490,25 +490,30 @@ You can find more information on how to use it in the corresponding GitHub repo.
 
 For more information about using the metrics with Prometheus and Grafana, see <<_metrics>>
 
-===== Logging
-The `logging` field allows the configuration of loggers. These loggers are:
-[source]
-log4j.rootLogger
-connect.root.logger.level
-log4j.logger.org.apache.zookeeper
-log4j.logger.org.I0Itec.zkclient
-log4j.logger.org.reflections
+===== [[logging_examples]]Logging
+The `logging` field allows the configuration of loggers. These loggers for zookeeper and kafka are available in `spec.zookeeper.logging` and `spec.kafka.logging` sections above respectively.
```

Comment:
Could we add a link to the `spec.zookeeper.logging` and `spec.kafka.logging` descriptions? I had a bit trouble finding it.